README.md

```diff
@@ -67,14 +101,36 @@ Project aims to achieve:
 
 ### KYODO features
 * Create a working DAO group. KYODO provides a clear interface for manage processes, gives a constructor to build a customized version of a DAO for the wide range of spheres.
-* Ability to mint your own token or use existing mintable one. We use Colony to own and manage it.
+* Ability to mint your own token or use existing mintable one. Based on Colony.
 * Instant tipping tool - instant payment for a contribution in crypto via different working chats. DF balance is reflected on the KYODO interface, and kept on Ethereum blockchain.
 * Token economy with inflation as an incentive system.
-* Democratic voting on the blockchain.
+* Democratic voting on the blockchain. Based on Aragon.
 * Multi-signature wallet holding fund treasury.
 * Task management interface.
 * Network effect incorporated.
 
+### Domains
+All the activities of DAOs could be divided within domains, that reflect it's specifics. 
+For example Decent.Fund has 4 domains:
+* DEV domain (UI/UX design, programming, digital marketing)
+* SOCIAL domain (research, articles, infographics, meetups, hackathons, social activity)
+* FUND management domain (smart investment, crypto asset management)
+* GOV domain (governance strategy)
+
+![](https://i.imgur.com/KSi8J3Y.jpg)
+
+The domain concept KYODO takes from Colony. 
```

Comment:
I wouldn't say that we've taken domain concept from Colony as it happened naturally. We might say that we extend Colony domain concept